### ISSUE TITLE
manager::tests::test_register() fails randomly

### ISSUE THREAD

**Issue by andreeaflorescu** [rust-vmm/event-manager#29]
Output:

```
failures:
--
  |  
  | ---- manager::tests::test_register stdout ----
  | thread 'manager::tests::test_register' panicked at 'called `Result::unwrap()` on an `Err` value: Epoll(Error(9))', src/libcore/result.rs:1165:5
```

Buildkite link: https://buildkite.com/rust-vmm/event-manager/builds/47#5a08d599-fa7a-4134-9ef6-7c313e9f4cbf

**Comment by jiangliu** [comment]
It always passes on my local dev machine:)

**Comment by andreeaflorescu** [comment]
@lauralt did some more digging, and there are multiple tests that are failing as per: https://github.com/rust-vmm/event-manager/pull/58#issuecomment-766710010

**Comment by lauralt** [comment]
Fixed by #63.
